### PR TITLE
Feature/remove far games

### PR DESCRIPTION
**Pull request by ArnaudPoletto**
Remove games that are too far away from the user in the game list.

### CODE REVIEW

**Review by RafaelRamirezS — APPROVED** [review]
Looks good to me

File: app/src/main/java/sdp/moneyrun/map/MapPlayerListAdapter.java [hunk 1]
```diff
@@ -19,7 +19,7 @@ public MapPlayerListAdapter(Activity context, ArrayList<Player> playerList) {
     }
 
     public View getView(int position, View view, ViewGroup parent) {
-        view = LayoutInflater.from(getContext()).inflate(R.layout.leaderboard_item_layout, null, true);
+        view = LayoutInflater.from(getContext()).inflate(R.layout.leaderboard_item_layout, null, false);
         Player player = getItem(position);
```
Comment (Alvaro2112) [inline app/src/main/java/sdp/moneyrun/map/MapPlayerListAdapter.java:23]: Is this the focusable parameter? The one you set from true to false?

File: app/src/main/java/sdp/moneyrun/menu/JoinGameImplementation.java [hunk 2]
```diff
@@ -53,14 +55,16 @@ public JoinGameImplementation(Activity activity,
                                   ActivityResultLauncher<String[]> requestPermissionsLauncher,
                                   FusedLocationProviderClient fusedLocationClient,
                                   boolean focusable,
-                                  int layoutId){
+                                  int layoutId,
+                                  boolean isLocationMocked){
```
Comment (Alvaro2112) [inline app/src/main/java/sdp/moneyrun/menu/JoinGameImplementation.java:59]: There should not be mock location attributes in non-test files.

File: app/src/main/java/sdp/moneyrun/ui/menu/MenuActivity.java [hunk 3]
```diff
@@ -126,13 +132,16 @@ public void runFunctionalities() {
         boolean guestPlayer = getIntent().getBooleanExtra("guestPlayer", false);
         setGuestPlayerFields(guestPlayer);
 
+        boolean isLocationMocked = getIntent().getBooleanExtra("isLocationMocked", false);
```
Comment (Alvaro2112) [inline app/src/main/java/sdp/moneyrun/ui/menu/MenuActivity.java:135]: Same here, mock location attributes don't belong in non-test files.

**Review by Alvaro2112 — CHANGES_REQUESTED** [review]

File: app/src/main/java/sdp/moneyrun/map/MapPlayerListAdapter.java (hunk 1 above)
Comment (ArnaudPoletto) [reply app/src/main/java/sdp/moneyrun/map/MapPlayerListAdapter.java:23]: No, it is the attachToRoot parameter. If the root is null, it is better to set it to false.

File: app/src/main/java/sdp/moneyrun/menu/JoinGameImplementation.java (hunk 2 above)
Comment (ArnaudPoletto) [reply app/src/main/java/sdp/moneyrun/menu/JoinGameImplementation.java:59]: This is the only way I managed to make the tests pass on Cirrus. This parameter can only be set to true via a putExtra.

File: app/src/main/java/sdp/moneyrun/ui/menu/MenuActivity.java (hunk 3 above)
Comment (ArnaudPoletto) [reply app/src/main/java/sdp/moneyrun/ui/menu/MenuActivity.java:135]: Should I change this and rethink the whole test mechanism?

File: app/src/main/java/sdp/moneyrun/map/MapPlayerListAdapter.java (hunk 1 above)
Comment (Alvaro2112) [reply app/src/main/java/sdp/moneyrun/map/MapPlayerListAdapter.java:23]: Ok, I see, good catch!

File: app/src/main/java/sdp/moneyrun/ui/menu/MenuActivity.java (hunk 3 above)
Comment (Alvaro2112) [reply app/src/main/java/sdp/moneyrun/ui/menu/MenuActivity.java:135]: I think so yeah, I will have a look at it this afternoon to see if I can come up with a solution

Comment (ArnaudPoletto) [reply app/src/main/java/sdp/moneyrun/ui/menu/MenuActivity.java:135]: Thank you

**Pull request closed without merging**